src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,38 @@
 'use strict';
 
 class Animal {
-  // write your code here
-}
 
+
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
+}
+Animal.alive = [];
```

Comment:
I think, you should first check whether this array exists and then push `this`

Reply:
That's because lint doesn't pass tests. This is a lint workaround.